core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -995,6 +996,30 @@ private[spark] class AppStatusListener(
     }
   }
 
+  def updateBroadcastBlock(event: SparkListenerBlockUpdated, broadcast: BroadcastBlockId): Unit = {
```

Comment:
`def` -> `private def`?

Reply:
Thank you for your review! I have submitted a new change.